disnake/app_commands.py

```diff
@@ -245,16 +245,16 @@ def __init__(
         self,
         name: LocalizedRequired,
         description: LocalizedOptional = None,
-        type: Union[OptionType, int] = None,
+        type: Optional[Union[OptionType, int]] = None,
         required: bool = False,
-        choices: Choices = None,
-        options: list = None,
-        channel_types: List[ChannelType] = None,
+        choices: Optional[Choices] = None,
+        options: Optional[list] = None,
```

Comment:
out of scope, but could this be more specific?

Reply:
Definitely, I've got another branch somewhere that fixes these across the entire lib. Out of scope though, yeah.